src/Upgrade/Infrastructure/PackageManager/CommandExecutor/ComposerCommandExecutor.php

```diff
@@ -87,6 +87,22 @@ public function __construct(ProcessRunnerServiceInterface $processRunner, Config
         $this->configurationProvider = $configurationProvider;
     }
 
+    /**
+     * @param \Upgrade\Domain\Entity\Collection\PackageCollection $packageCollection
+     *
+     * @return \Upgrade\Application\Dto\PackageManagerResponseDto
+     */
+    public function updateSubPackages(PackageCollection $packageCollection): PackageManagerResponseDto
+    {
+        $command = explode(' ', sprintf(
+            '%s%s',
+            static::UPDATE_COMMAND_NAME,
+            $this->getPackageString($packageCollection),
```

Comment:
do we need additional flags here, such as `--no-plugins`, `--no-install`, `--with-dependencies`, etc ?

Reply:
no